beetsplug/bpd/gstplayer.py

```diff
@@ -27,7 +27,11 @@
 
 from beets import ui
 
-gi.require_version("Gst", "1.0")
+try:
```

Comment:
Could you add a comment here on why we reraise as ImportError? If I come around this in the wild I would be curious on what the reason for this is.

Reply:
Perfectly sensible!

Reply:
This is more than enough, thanks! 

I would probably remove the specific mention of the `test_bpd.py` file, as these things tend to drift apart over time but it should be fine in the grand scheme.